bindings_ffi/src/mls.rs

```diff
@@ -318,6 +320,39 @@ pub struct FfiGroup {
     inner_client: Arc<RustXmtpClient>,
     group_id: Vec<u8>,
     created_at_ns: i64,
+    added_by: Option<FfiGroupMember>,
+}
+
+pub fn create_ffi_group(
```

Comment:
You shouldn't be creating a new function. It should just be added to the existing function https://github.com/xmtp/libxmtp/blob/main/bindings_ffi/src/mls.rs#L195-L227

Reply:
Per conversation we're going to roll this back this commit `3c3d9f363400fb35a75165c362773cdd2ecaf3d6`